src/dialects/mysql/data-types.js

```diff
@@ -6,6 +6,7 @@ const momentTz = require('moment-timezone');
 const moment = require('moment');
 
 module.exports = BaseTypes => {
+  const warn = BaseTypes.ABSTRACT.warn.bind(undefined, 'https://www.sqlite.org/datatype3.html');
```

Comment:
Same here, this is the proper link for MySQL:

```suggestion
  const warn = BaseTypes.ABSTRACT.warn.bind(undefined, 'https://dev.mysql.com/doc/refman/8.0/en/data-types.html');
```